### ISSUE TITLE
Running WPS/WRF program in Python 3.7 raises OSError: [WinError 87] The parameter is incorrect

### ISSUE THREAD

**Issue by dmey** [GIS4WRF/gis4wrf#101]
I am preparing the release to support QGIS 3.4.1 (see branch [0.13.2](https://github.com/GIS4WRF/gis4wrf/tree/0.13.2)) however, when I try to launch a program I get the following:

```
An error has occurred while executing Python code: 

OSError: [WinError 87] The parameter is incorrect 
Traceback (most recent call last):
  File "C:/Users/testing/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\gis4wrf\plugin\ui\widget_run.py", line 301, in on_finished
    raise thread.exc_info[0].with_traceback(*thread.exc_info[1:])
  File "C:/Users/testing/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\gis4wrf\plugin\ui\thread.py", line 85, in run
    self.use_mpi, self.mpi_processes):
  File "C:/Users/testing/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\gis4wrf\core\program.py", line 57, in run_program
    mpi_path = find_mpiexec()
  File "C:/Users/testing/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\gis4wrf\core\program.py", line 39, in find_mpiexec
    subprocess.check_output([path] + help_option, startupinfo=STARTUPINFO)
  File "C:\PROGRA~1\QGIS3~1.4\apps\Python37\lib\subprocess.py", line 376, in check_output
    **kwargs).stdout
  File "C:\PROGRA~1\QGIS3~1.4\apps\Python37\lib\subprocess.py", line 453, in run
    with Popen(*popenargs, **kwargs) as process:
  File "C:\PROGRA~1\QGIS3~1.4\apps\Python37\lib\subprocess.py", line 756, in __init__
    restore_signals, start_new_session)
  File "C:\PROGRA~1\QGIS3~1.4\apps\Python37\lib\subprocess.py", line 1155, in _execute_child
    startupinfo)
OSError: [WinError 87] The parameter is incorrect


Python version: 3.7.0 (v3.7.0:1bf9cc5093, Jun 27 2018, 04:59:51) [MSC v.1914 64 bit (AMD64)] 
QGIS version: 3.4.1-Madeira Madeira, 383851c597 

Python Path:
C:/PROGRA~1/QGIS3~1.4/apps/qgis/./python
C:/Users/testing/AppData/Roaming/QGIS/QGIS3\profiles\default/python
C:/Users/testing/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins
C:/PROGRA~1/QGIS3~1.4/apps/qgis/./python/plugins
C:\Program Files\QGIS 3.4\bin\python37.zip
C:\PROGRA~1\QGIS3~1.4\apps\Python37\DLLs
C:\PROGRA~1\QGIS3~1.4\apps\Python37\lib
C:\Program Files\QGIS 3.4\bin
C:\PROGRA~1\QGIS3~1.4\apps\Python37
C:\PROGRA~1\QGIS3~1.4\apps\Python37\lib\site-packages
C:\PROGRA~1\QGIS3~1.4\apps\Python37\lib\site-packages\win32
C:\PROGRA~1\QGIS3~1.4\apps\Python37\lib\site-packages\win32\lib
C:\PROGRA~1\QGIS3~1.4\apps\Python37\lib\site-packages\Pythonwin
C:/Users/testing/AppData/Roaming/QGIS/QGIS3\profiles\default/python
C:\Users\testing\AppData\Local\gis4wrf\python37
C:\Users\testing\AppData\Local\gis4wrf\python37\lib\site-packages
```

**Comment by letmaik** [comment]
Weird. Not setting startupinfo makes it work, but the point of using it 
is to hide the console window. Need to reproduce on a standard Python 
3.7 installation outside QGIS. If it's the same, then we may have found 
a Python bug, but I doubt it.

**Comment by letmaik** [comment]
Turns out it's a bug in Python 3.7.0 that was fixed in Python 3.7.1: https://bugs.python.org/issue34044

The work-around described in that issue is to not re-use startupinfo objects but create them from scratch for each subprocess call. I verified this and it works. I'll fix it in this branch.

**Comment by dmey** [comment]
Fixed in faef1a5b4f9845c5b83810e8d3aa79392d464a8a and c36aae0d1b46d7bae59882bcbe3e5fff9e9858e6
